template/base/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "vite": "^5.2.11"
+  },
+  "dependencies": {
+    "@vitejs/plugin-vue": "^5.0.5"
+  }
```

Comment:
Do we need it?

Reply:
yes we need it because when i remove this project crash